src/bindings/c/tests/ov_multithreading_test.cpp

```diff
@@ -0,0 +1,162 @@
+// Copyright (C) 2018-2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#include <thread>
+
+#include "ov_test.hpp"
+
+class ov_multithreading_test : public ov_capi_test_base {
+    void SetUp() override {
+        ov_capi_test_base::SetUp();
+    }
+
+    void TearDown() override {
+        ov_capi_test_base::TearDown();
+    }
+
+protected:
+    unsigned int iterations;
+    unsigned int threadsNum;
```

Comment:
```suggestion
    unsigned int threads_num;
```
Pls keep the same naming style